Makefile

```diff
@@ -56,5 +56,15 @@ test: test-unit ## Run the tests
 test-unit: ## Run the unit tests
 	$(Q)go test -count=1 -short ./...
 
+deploy: generate ## Deploy the Combo operator to the current cluster
+	kubectl apply -f manifests/crds			  && \
+	kubectl apply -f manifests/namespace.yaml && \
+	kubectl apply -f manifests/deploy
+
+teardown: ## Teardown the Combo operator to the current cluster
+	kubectl delete -f manifests/deploy		   && \
+	kubectl delete -f manifests/namespace.yaml && \
+	kubectl delete -f manifests/crds
+
 # Binary builds
-GO_BUILD := $(Q)go build
+GO_BUILD := $(Q)go build
```

Comment:
Nit: Newline. This advice applies in all instances, the reasoning behind this advice can be found [here](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).